assets/app/styles/_components.less

```diff
@@ -83,6 +83,7 @@
           overflow: hidden;
           text-overflow: ellipsis;
           line-height: 1.3;
+          white-space: nowrap;
```

Comment:
I think this is way too broad of a "fix"... needs to be targeted to the port header specifically, if we go the nowrap route